src/graphql/trusted_domain.rs

```diff
@@ -61,6 +62,33 @@ impl TrustedDomainMutation {
         Ok(name)
     }
 
+    /// Update a trusted domain, returning the new value if it passes domain validation.
+    #[graphql(guard = "RoleGuard::new(Role::SystemAdministrator)
+        .or(RoleGuard::new(Role::SecurityAdministrator))")]
+    async fn update_trusted_domain(
+        &self,
+        ctx: &Context<'_>,
+        old: TrustedDomainInput,
+        new: TrustedDomainInput,
+    ) -> Result<String> {
+        if !is_valid_domain(&new.name) {
+            return Err(TrustedDomainError::InvalidDomainName(String::from(&new.name)).into());
+        }
+
+        let name = {
+            let store = crate::graphql::get_store(ctx).await?;
+            let map = store.trusted_domain_map();
+            let old = review_database::TrustedDomain::from(old);
+            let new = review_database::TrustedDomain::from(new);
+            map.update(&old, &new)?;
+            new.name
+        };
+
+        let agent_manager = ctx.data::<BoxedAgentManager>()?;
```

Comment:
@sophie-cluml 
It is necessary to determine whether it is important to use broadcasts.
It doesn't seem to be used in the source in review-web, but I think if it's an unnecessary source, it's better to remove it.

Reply:
If I am not mistaken, I think we discussed this over an online meeting, and the conclusion was that we will use `broadcast_trusted_domains()`. I believe that discussion was quite a while ago, so I would like to know if we are on the same page. Please let me know.